scalableminds/webknossos
# Add REST API versioning support (second try)

Reverts scalableminds/webknossos#3404

### Steps to test:
- check that both /api/buildinfo and /api/v1/buildinfo reply
- check that standalone datastore setup still works (will update guide after this PR is merged)

### Changes to standalone datastore setup
add these three lines to the end of your application.conf (no changes needed in routes file anymore)
```
play.modules.disabled += "com.scalableminds.webknossos.datastore.DataStoreModule"
datastore.enabled = false
play.http.router = "noDS.Routes"
```

### Changes necessary in kube
- `play.http.router` should be changed to `"noDS.noTS.Routes"` for mhlab, as they don’t have a local datastore nor tracingstore. see Migrations.md

------
- ~~[] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
